jekyll/_sass/_subnav.scss

```diff
@@ -0,0 +1,31 @@
+// The following variables required by shared subnav component.
+// Values are copied from circleci.com project.
+$border-width: 1px;
+$border-width-widest: 4px;
+$z-index-subnav: 2;
+$gray: #D8D8D8;
+$white: #FFFFFF;
+$text-color-header: $black-light;
+$transition-duration-medium: 400ms;
+
+// These values are also required, but have been customized to match the nav height of this project.
+$nav-height: 120px;
+$subnav-height: 64px;
+$nav-offset: $nav-height;
+$subnav-offset: $subnav-height;
+$nav-offset-subnav: $nav-offset + $subnav-offset;
+
+@import 'shared/components/subnav';
+
+// These tweaks to the shared subnav CSS are necessary to work in the context of this project's layout.
+
+#subnav-placeholder,
+.subnav.component.js-fixed {
+  width: inherit;
+}
+
+.deep-link-anchor {
+  &.subnav-offset {
+    top: -$nav-offset-subnav;
+  }
+}
```

Comment:
The first list item is shown as flush left in the design.

```css
.component.subnav li:first-child {
  margin-left: 0;
}
```

Before:
![Screen Shot 2020-10-09 at 11 20 06 AM](https://user-images.githubusercontent.com/5113432/95601613-b31aab00-0a21-11eb-9568-f9ef054e9551.png)

After:
![Screen Shot 2020-10-09 at 11 21 39 AM](https://user-images.githubusercontent.com/5113432/95601633-b6ae3200-0a21-11eb-9461-6a50968e64a7.png)

Additionally, the design has everything (search input, first subnav item, and results) all with the same left alignment, but more indented (relative to search icon placement) than what's here.

Design:
![Screen Shot 2020-10-09 at 9 42 11 AM](https://user-images.githubusercontent.com/5113432/95601763-ebba8480-0a21-11eb-8435-9cdecb0e1d4d.png)

Reply:
Oh also, the non-active items in the subnav in the design are bolded, but gray; 
rather than a regular weight and black(ish)